04week/loop.js

```diff
@@ -0,0 +1,68 @@
+'use strict'
+
+// for loop
+// Use a for loop to console.log each item in the array carsInReverse.
+
+const carsInReverse = ['ford', 'nissan', 'honda', 'BMW'];
+
+
+for(let i = 0; i < carsInReverse.length; i ++){
```

Comment:
function should reverse the order of the car array, If your having problems figuring it out think of the index of the values.